setup.sh

```diff
@@ -165,6 +166,27 @@ function cmd_output_manager()
   return "$?"
 }
 
+# KW used git to track saved configs from kernel-config-manager.
+# It changed so this function removes the unused .git folder from
+# the dot_configs_dir
+function remove_kernel_config_manager_legacy_git()
+{
+  local -r original_path="$PWD"
+
+  [[ ! -d "$dot_configs_dir"/.git ]] && return
+
+  if pushd "$dot_configs_dir"; then
+    rm -rf .git/
+    popd || {
+      complain "Could not return to original path from dot_configs_dir"
+      exit 1
+    }
+  else
+    complain 'Could not cd to dot_configs_dir'
```

Comment:
Add the `dot_configs_dir` value as part of this complain; this can be useful for debugging.